### PR TITLE
doc: add Chinese faq and fix typo

### PR DESCRIPTION
**Pull request by wzhudev**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you propose PR to right branch: bugfix for `master`, feature for branch `feature`.
* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

**Comment by wendzhue** [comment]
@afc163 Please review this.

File: docs/react/faq.zh-CN.md [hunk 1]
```diff
@@ -3,115 +3,113 @@ order: 9
 title: FAQ
 ---
 
-Search in the questions about React first: https://reactfaq.site .
+有问题请先问 React：https://reactfaq.site 。
 
 ---
 
-### Are you going to provide Vue(etc...) edition?
+### 你们会提供 Vue 等版本吗？
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: 提供 Vue 版本的 Ant Design 么？

File: docs/react/faq.zh-CN.md [hunk 2]
```diff
@@ -3,115 +3,113 @@ order: 9
...
-Use `<Select getPopupContainer={trigger => trigger.parentNode}>` to render component inside Popover. (Or other getXxxxContainer props)
+使用 `<Select getPopupContainer={trigger => trigger.parentNode}>` 来在 Popover 中渲染组件，或者其他 getXxxxContainer 参数。
 
 https://ant.design/components/select/#Select-props
 
-related issue: [#3487](https://github.com/ant-design/ant-design/issues/3487) [#3438](https://github.com/ant-design/ant-design/issues/3438)
+相关 issue：[#3487](https://github.com/ant-design/ant-design/issues/3487) [#3438](https://github.com/ant-design/ant-design/issues/3438)
 
-### `Select Dropdown DatePicker TimePicker Popover Popconfirm` will scroll with page?
+### `Select Dropdown DatePicker TimePicker Popover Popconfirm` 会滚动页面？
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: 会跟随滚动条上下移动

File: docs/react/faq.zh-CN.md [hunk 3]
```diff
@@ -3,115 +3,113 @@ order: 9
...
 
-Use `<Select getPopupContainer={trigger => trigger.parentNode}>` to render component inside the scroll area. (Or other getXxxxContainer props).
+使用 `<Select getPopupContainer={trigger => trigger.parentNode}>` 来渲染滚动区域内的组件，或者其他 getXxxxContainer 参数。
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]:
```diff
- 来渲染滚动区域内的组件
+ 来将组件渲染到滚动区域内
```

File: docs/react/faq.zh-CN.md [hunk 4]
```diff
@@ -3,115 +3,113 @@ order: 9
...
+参考：https://ant.design/docs/react/customize-theme 。
 
-### Would you supply other themes?
+### 你们会提供其他主题吗？
 
-No, we follow Ant Design specification. https://github.com/ant-design/ant-design/issues/1241
+不，我们遵守 Ant Design 设计规范。https://github.com/ant-design/ant-design/issues/1241 。
 
-### How to modify `Menu`/`Button`(etc...)'s style?
+### 如何修改 `Menu`/`Button` 等等的样式？
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]:
```diff
- 如何修改 `Menu`/`Button` 等等的样式？
+ 如何修改 Ant Design 组件的默认样式？
```

File: docs/react/faq.zh-CN.md [hunk 5]
```diff
@@ -3,115 +3,113 @@ order: 9
...
 
-You can override its style. But we don't recommend to do so. antd is not only a set of React components but also a design specification.
+你可以覆盖它们的样式，但是我们不推荐这么做。antd 是一系列 React 组件，但同样是一套设计规范。
 
-### I just want to use `Menu`/`Button`(etc...), but it seems that I have to import the whole antd and its style.
+### 我只想使用 `Menu`/`Button` 等，但似乎我必须 import 整个 antd 和它的样式文件。
 
-Try [babel-plugin-import](https://github.com/ant-design/babel-plugin-import), or import what you need in this way:
+试试 [babel-plugin-import](https://github.com/ant-design/babel-plugin-import)，或者用下面这种方式 import 你想要的：
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]:
```diff
-  import 你想要的
+ 按需加载
```

File: docs/react/faq.zh-CN.md [hunk 6]
```diff
@@ -3,115 +3,113 @@ order: 9
...
+参考：https://github.com/jmblog/how-to-optimize-momentjs-with-webpack 。
 
-### It doesn't work when I change `defaultValue` dynamically.
+### 当我动态改变 `defaultValue` 的时候它并没有生效。
 
-The `defaultXxxx` (like `defaultValue`) of `Input`/`Select`(etc...) only works in first render. It is a specification of React, please read [React's documentation](https://facebook.github.io/react/docs/forms.html#controlled-components).
+ `Input`/`Select` 等的 `defaultXxxx`（例如 `defaultValue`）只有在第一次渲染的时候有效，这是 React 的规范，请阅读 [React 的文档](https://facebook.github.io/react/docs/forms.html#controlled-components)。
 
-### I set the `value` of `Input`/`Select`(etc...), and then, it cannot be changed by user's action.
+### 当我设置了 `Input`/`Select` 等的 `value`，它不能被用户的操作所改变。
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: 略有点太翻译腔了。😀

『它不能被用户的操作所改变』改成『时，就无法修改了』

File: docs/react/faq.zh-CN.md [hunk 7]
```diff
@@ -3,115 +3,113 @@ order: 9
...
 
-Try `defaultValue` or `onChange` to change `value`, and please read [React's documentation](https://facebook.github.io/react/docs/forms.html#controlled-components).
+尝试使用 `defaultValue` 和 `onChange` 来改变 `value`，并阅读 [React 的文档](https://facebook.github.io/react/docs/forms.html#controlled-components)。
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: 和 => 或

File: docs/react/faq.zh-CN.md [hunk 8]
```diff
@@ -3,115 +3,113 @@ order: 9
...
+### antd 覆盖了我的全局样式！
 
-Yes, antd is designed to develop a complete background application, we override some global styles for styling convenience and it can't be removed now. More info trace https://github.com/ant-design/ant-design/issues/4331 .
+是的，antd 在设计的时候就是用来开发一个完整的应用的，为了方便，我们覆盖了一些全局样式，现在还不能移除，想要了解更多请追踪这个 issue：https://github.com/ant-design/ant-design/issues/4331 ，或者参考这个教程 [How to avoid modifying global styles?](docs/react/customize-theme#How-to-avoid-modifying-global-styles-?)
 
-Or follow instruction in [How to avoid modifying global styles?](docs/react/customize-theme#How-to-avoid-modifying-global-styles-?)
+### 我没法安装 `antd` 和 `antd` 的依赖，顺便提一句，我在中国大陆。
 
-### I cannot install `antd` and `antd`'s dependencies(etc...). FYI, I live in China mainland.
+那啥，试试 [cnpm](http://npm.taobao.org/)。
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: 👍

File: docs/react/faq.zh-CN.md [hunk 9]
```diff
@@ -3,115 +3,113 @@ order: 9
...
 
-Long live the Girl Friend Wall! And try [cnpm](http://npm.taobao.org/).
+### 我在 package.json 中将 `dependencies.antd` 添加为 git repository，但是没有用。
 
-### I set `dependencies.antd` as git repository in package.json, but it doesn't work.
+当然了。请使用 npm 安装 `antd`。
 
-Yes, please install `antd` with npm.
+### `message` 和 `notification` 是小写的，但是其他的组件都是首字母大写的，这是手滑吗？
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: 😀

File: docs/react/faq.zh-CN.md (hunk 8 above)
Comment (zombieJ) [reply docs/react/faq.zh-CN.md]: 哈哈，这个有趣

File: docs/react/faq.zh-CN.md [hunk 10]
```diff
@@ -3,115 +3,113 @@ order: 9
...
 
-### `message` and `notification` is lower case, but other components are capitalized. Typo?
+不，因为 `message` 是一个函数，而不是一个 React 组件。
 
-No, as `message` is just a function, not a React Component.
+### `antd` 在移动端体验不佳。
 
-### `antd` doesn't work well in mobile.
+请浏览 [And Design Mobile](http://mobile.ant.design) 以了解详情。`antd` 并没有被优化以适应移动端。你可以试试 [react-component](https://github.com/react-component/)，其中带有 'm-' 'rn-' 前缀的库是为移动端设计的。
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]:
```diff
- 并没有被优化以适应移动端
+ 并非针对移动端设计
```

File: docs/react/faq.zh-CN.md [hunk 11]
```diff
@@ -121,40 +119,40 @@ Here are some typical wrong examples:
...
+### 你们有渠道来接受捐助吗，比如 PayPal 和支付宝？
 
-No yet.
+还没有。
 
-### Why not?
+### 为什么？
 
-Alibaba will pay us.
+马爸爸会付给我们钱。
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: yes!

**Comment by afc163** [comment]
@wendzhue 特别赞，加了一些 review 意见。翻译腔略重了点，可以再整体调整调整。

File: docs/react/faq.zh-CN.md [hunk 12]
```diff
@@ -119,7 +119,7 @@ import { Menu, Breadcrumb, Icon } from 'antd';
 - ❌ antdesign
 - ❌ Antdesign
 
-### 你们有渠道来接受捐助吗，比如 PayPal 和支付宝？
+### 你们有接受捐助的渠道吗，比如 PayPal 或者支付宝？
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: 中文语境 PayPal 可以换成微信，然后把支付宝放前面。

File: docs/react/faq.zh-CN.md [hunk 13]
```diff
@@ -64,11 +64,11 @@ import { Menu, Breadcrumb, Icon } from 'antd';
...
 ### 当我动态改变 `defaultValue` 的时候它并没有生效。
 
- `Input`/`Select` 等的 `defaultXxxx`（例如 `defaultValue`）只有在第一次渲染的时候有效，这是 React 的规范，请阅读 [React 的文档](https://facebook.github.io/react/docs/forms.html#controlled-components)。
+`Input`/`Select` 等的 `defaultXxxx`（例如 `defaultValue`）只有在第一次渲染的时候有效，这是 React 的规范，请阅读 [React 的文档](https://facebook.github.io/react/docs/forms.html#controlled-components)。
 
-### 当我设置了 `Input`/`Select` 等的 `value`，它不能被用户的操作所改变。
+### 当我设置了 `Input`/`Select` 等的 `value` 时它就无法修改了。
 
-尝试使用 `defaultValue` 和 `onChange` 来改变 `value`，并阅读 [React 的文档](https://facebook.github.io/react/docs/forms.html#controlled-components)。
+尝试使用 `defaultValue` 或 `onChange` 来改变 `value`，请参考 [React 的文档](https://facebook.github.io/react/docs/forms.html#controlled-components)。
```
Comment (afc163) [inline docs/react/faq.zh-CN.md]: 这个链接可以换成 https://reactjs.org/docs/forms.html#controlled-components

Comment (wendzhue) [reply docs/react/faq.zh-CN.md]: Will do~

**Pull request merged**